README.md

```diff
@@ -1,306 +1,184 @@
+Running a Local Geth Node
+------------------------------
 
-The above test network is a cross client one based on the ethash proof-of-work consensus algorithm. As such, it has certain extra overhead and is more susceptible to reorganization attacks due to the network's low difficulty / security. Go Ethereum also supports connecting to a proof-of-authority based test network called [*Rinkeby*](https://www.rinkeby.io) (operated by members of the community). This network is lighter, more secure, but is only supported by go-ethereum.
+To start a local Geth node, you can create your own `genesis.json` file similar to:
 
+```json
+{
+    "config": {
+        "chainId": 12345,
+        "homesteadBlock": 0,
+        "eip155Block": 0,
+        "eip158Block": 0
+    },
+    "difficulty": "200",
+    "gasLimit": "210000000000"
+}
 ```
-$ geth --rinkeby console
-```
-
-### Configuration
 
-As an alternative to passing the numerous flags to the `geth` binary, you can also pass a configuration file via:
+Then, you can build `geth` and init a new instance of a local, Ethereum blockchain as follows:
 
 ```
-$ geth --config /path/to/your_config.toml
+$ make geth
+$ ./build/bin/geth init ./sharding/genesis.json -datadir /path/to/your/datadir
+$ ./build/bin/geth --nodiscover console --datadir /path/to/your/datadir --networkid 12345
 ```
-
-To get an idea how the file should look like you can use the `dumpconfig` subcommand to export your existing configuration:
+Then, the geth console can start up and you can start a miner as follows:
 
 ```
-$ geth --your-favourite-flags dumpconfig
+> personal.newAccount()
+> miner.setEtherbase(eth.accounts[0])
+> miner.start()
 ```
 
-*Note: This works only with geth v1.6.0 and above.*
-
-#### Docker quick start
-
-One of the quickest ways to get Ethereum up and running on your machine is by using Docker:
+Then, once you are satisfied with mining for a few seconds, stop the miner with
 
 ```
-docker run -d --name ethereum-node -v /Users/alice/ethereum:/root \
-           -p 8545:8545 -p 30303:30303 \
-           ethereum/client-go
+> miner.stop()
 ```
 
-This will start geth in fast-sync mode with a DB memory allowance of 1GB just as the above command does.  It will also create a persistent volume in your home directory for saving your blockchain as well as map the default ports. There is also an `alpine` tag available for a slim version of the image.
-
-Do not forget `--rpcaddr 0.0.0.0`, if you want to access RPC from other containers and/or hosts. By default, `geth` binds to the local interface and RPC endpoints is not accessible from the outside.
+Now, save the passphrase you used in the geth node into a text file called password.txt. Then, once you have this private geth node running on your local network, we will need to generate fake, pending transactions that can then be processed into shards by validators and proposers. For this, we have created an in-house transaction generator CLI tool.
```

Comment:
s/fake/test/. Please call these test transactions because they are "real", but only serve the purposes of testing.